sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1427,6 +1427,20 @@ object functions {
    */
   def acos(columnName: String): Column = acos(Column(columnName))
 
+  /**
+   * @return inverse hyperbolic cosine of `e`
+   *
+   * @group math_funcs
+   */
```

Comment:
We should probably have `@since` annotation here and for the remaining ones.

Reply:
I wasn't sure whether this is something I should put in during the pull-request, or whether it gets added at a later stage.

Shall I presume that this is destined for Spark-3.1, or maybe 3.0.2?

Reply:
It would be 3.1, as new functions are not added in maintenance releases.